### DISCUSSION TITLE
The second ADC channel can't be used

### DISCUSSION THREAD

**Discussion by luciangpl** [arendst/Tasmota#12785] [Ideas]
Hi, 
I know that the second ADC channel can't be used together with Wi-Fi, but I saw there is a workaround, would it be possible to apply it? (espressif/arduino-esp32#440)
And another request, at this moment, the maximum number of inputs used as "ADC input" is 8, can you increase that number to 12?
![image](https://user-images.githubusercontent.com/10548021/127605492-0881e138-b599-49d0-9418-43c9c1fbdde0.png)

**Comment by Jason2866** [comment]
About ADC and wifi conflict. No change, since wifi has top priority for Tasmota and has to work always.
Increasing from 8 to 12 is not an easy change. Will probably not done.

**Reply by luciangpl** [reply]
In that workaround, the WIFI is disabled for a very short period of time, just enough to get the ADCs measurements. 
It would be nice to have all 12 analog inputs available

**Comment by Jason2866** [comment]
I did some more research. It is a unstable (not for all chip revisions working) workaround.
Furthermore it needs a old IDF/Arduino build. Will **NOT** implemented in Tasmota.
